### ISSUE TITLE
SDXL training lora failed: size mismatch 

### ISSUE THREAD

**Issue by bakkhos8** [kohya-ss/sd-scripts#761]
I can train lora by SD 1.5 successfully, but when I try to use SDXL base , error msg is 

> 
> size mismatch for down_blocks.1.attentions.0.proj_in.weight: copying a param with shape torch.Size([640, 640]) from checkpoint, the shape in current model is torch.Size([640, 640, 1, 1]).
> 	size mismatch for down_blocks.1.attentions.0.transformer_blocks.0.attn2.to_k.weight: copying a param with shape torch.Size([640, 2048]) from checkpoint, the shape in current model is torch.Size([640, 768]).
> 	size mismatch for down_blocks.1.attentions.0.transformer_blocks.0.attn2.to_v.weight: copying a param with shape torch.Size([640, 2048]) from checkpoint, the shape in current model is torch.Size([640, 768]).
> 
> ......
> 
> '['/home/deploy/lora-scripts-v1.5.0/venv/bin/python', './sd-scripts/train_network.py', '--enable_bucket', '--pretrained_model_name_or_path=./sd-models/sd_xl_base_1.0.safetensors', '--train_data_dir=./train/project', '--output_dir=./output', '--logging_dir=./logs', '--log_prefix=trained', '--resolution=640,960', '--network_module=networks.lora', '--max_train_epochs=15', '--learning_rate=1e-4', '--unet_lr=1e-4', '--text_encoder_lr=1e-5', '--lr_scheduler=cosine_with_restarts', '--lr_warmup_steps=0', '--lr_scheduler_num_cycles=1', '--network_dim=32', '--network_alpha=32', '--output_name=trained', '--train_batch_size=3', '--save_every_n_epochs=2', '--mixed_precision=fp16', '--save_precision=fp16', '--seed=1337', '--cache_latents', '--prior_loss_weight=1', '--max_token_length=225', '--caption_extension=.txt', '--save_model_as=safetensors', '--min_bucket_reso=256', '--max_bucket_reso=1024', '--keep_tokens=0', '--xformers', '--shuffle_caption', '--v2', '--v_parameterization', '--optimizer_type', 'Lion', '--log_with=tensorboard']' returned non-zero exit status 1.
> 

both --v2 and --v_parameterization is set

the SD model file is from https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0. 

picture size is 640x960, these pictures is trained to lora successful by using SD 1.5 base model.

**Comment by bakkhos8** [comment]
sorry, I just found the SDXL branch
